src/clr-addons/html-editor/html-editor.scss

```diff
@@ -0,0 +1,46 @@
+:root {
```

Comment:
what about adding this whole style directly to the phs theme so no further inclusion is necessary when using the phs theme? as you already use the phs colors here.
also we could apply the styles "magically" by selecting the html tag "angular-editor", so we don't have to apply a css class separately